docs/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.py

```diff
@@ -1,13 +1,19 @@
 """
-QBO Phase plots and QBO Index: UserScript, Stat-Analysis
-================================================================================
+UserScript and StatAnalysis: Compute QBO Phase plots and QBO Index
+==================================================================
 
 model_applications/
```

Comment:
```suggestion
model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.py
```
The [template](https://github.com/dtcenter/METplus/blob/develop/docs/use_cases/use_case_documentation_template.py) has the full path on one line.